docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -513,19 +513,17 @@ Sample Dockerfile for building with Maven:
 ----
 ## Stage 1 : build with maven builder image with native capabilities
 FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} AS build
-COPY --chown=quarkus:quarkus mvnw /code/mvnw
-COPY --chown=quarkus:quarkus .mvn /code/.mvn
-COPY --chown=quarkus:quarkus pom.xml /code/
-USER quarkus
-WORKDIR /code
+COPY pom.xml pom.xml
```

Comment:
Why did you remove the owner change?

Reply:
Also, do not use "/", it causes issues before (on some systems). You need to be under a new directoyr.